.github/workflows/release-cli.yml

```diff
@@ -174,19 +175,50 @@ jobs:
           cat target/homebrew/keboola-cli.rb
 
       - name: Upload Homebrew formulae
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
           name: homebrew-formulae
           path: target/homebrew/keboola-cli.rb
+          if-no-files-found: error
           retention-days: 1
 
       - name: Upload Scoop manifest
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
           name: scoop-manifest
-          path: target/keboola-cli.json
+          path: target/scoop/keboola-cli.json
+          if-no-files-found: error
           retention-days: 1
 
+  test-expected-files:
+    needs:
+      - version
+      - release
+    env:
+      VERSION: ${{ needs.version.outputs.VERSION }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check expected files exist
+        run: |
+          curl --output /dev/null --silent --head --fail "https://github.com/${{ github.repository }}/releases/download/v${{ env.VERSION }}/keboola-cli_${{ env.VERSION }}_checksums.txt"
+          curl --output /dev/null --silent --head --fail "https://github.com/${{ github.repository }}/releases/download/v${{ env.VERSION }}/keboola-cli_${{ env.VERSION }}_darwin_amd64.zip"
+          curl --output /dev/null --silent --head --fail "https://github.com/${{ github.repository }}/releases/download/v${{ env.VERSION }}/keboola-cli_${{ env.VERSION }}_darwin_arm64.zip"
+          curl --output /dev/null --silent --head --fail "https://github.com/${{ github.repository }}/releases/download/v${{ env.VERSION }}/keboola-cli_${{ env.VERSION }}_linux_amd64.apk"
+          curl --output /dev/null --silent --head --fail "https://github.com/${{ github.repository }}/releases/download/v${{ env.VERSION }}/keboola-cli_${{ env.VERSION }}_linux_amd64.deb"
+          curl --output /dev/null --silent --head --fail "https://github.com/${{ github.repository }}/releases/download/v${{ env.VERSION }}/keboola-cli_${{ env.VERSION }}_linux_amd64.rpm"
+          curl --output /dev/null --silent --head --fail "https://github.com/${{ github.repository }}/releases/download/v${{ env.VERSION }}/keboola-cli_${{ env.VERSION }}_linux_amd64.zip"
```

Comment:
A for loop would make it simpler.

https://linuxhint.com/bash_loop_list_strings/